client/src/app/shared/hooks/table-controls/filtering/getFilterHubRequestParams.ts

```diff
@@ -42,6 +42,13 @@ export const getFilterHubRequestParams = <
     // Note: If we need to support more of the logic operators in HubFilter in the future,
     //       we'll need to figure out how to express those on the FilterCategory objects
     //       and translate them here.
+    if (filterCategory.type === "numsearch") {
+      params.filters?.push({
+        field: categoryKey,
+        operator: "=",
+        value: Number(filterValue[0]),
+      });
+    }
```

Comment:
I'm surprised TS is okay with this since `value` of a `HubFilter` has to be a string or an object. We probably want to change the `HubFilter` type (at `client/src/app/api/models.ts`) so the value is `string | number` and also has `list: (string | number)[];` maybe? Also I left behind a TODO comment there that we should remove, saying we should remove HubFilter before I changed my mind 🙃

Reply:
Oh I totally missed that you fixed that